releasenotes/notes/always_send_trace_to_agent-f5dad70a40aa92a9.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an issue where previously if ``span.sampled`` was not set on a span 
+    in a trace, we would not send the trace to the agent. With this fix, all traces are sent to the agent.
```

Comment:
Can we simplify this release note. I don't think we need to include the detail about span.sampled. We should call out this could impact trace metrics.